dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/jaxrspropagation/responses/JaxRsResponseCodeTestServlet.java

```diff
@@ -82,7 +85,9 @@ public void testRedirect() {
         doTestForStatusCode(307);
     }
 
+    // Method followRedirects() was unavailable before RestClient 2.0
     @Test
+    @SkipForRepeat({TelemetryActions.MP14_MPTEL11_ID, TelemetryActions.MP41_MPTEL11_ID})
```

Comment:
Assuming the comment is correct, this test should pass with MP 4.1 which includes mpRestClient-2.0

```suggestion
    // Method followRedirects() was unavailable before RestClient 2.0
    @Test
    @SkipForRepeat({TelemetryActions.MP14_MPTEL11_ID})
```